corgi/core/models.py

```diff
@@ -651,7 +651,7 @@ def get_latest_components(self):
                         ),
                         el_match=Func(
                             F("release"),
-                            Value("el([0-9]+)"),
+                            Value(".([a-z]*el[6,7,8,9]+)"),
```

Comment:
If you're trying to match a literal dot . here, I think it needs to be \\. escaped so it isn't treated as part of the regex.

I also think using "[a-z]*" is a little dangerous, if we only want to match a single leading "a"...it feels like the approach here will lead to other side-effects or surprises.

If we want to constrain this to particular .el versions, I think the plus at the end can just be removed. Currently it will match version 66 - 69, 76 - 79, etc. as well as 6 - 9.

So that ends up as:
```suggestion
                            Value("\.(a?el[6-9])"),
```

Reply:
thx! safety eg. there was some weird el7#### ... but I guess we can layer on a fix for those if they ever become significant